backend/core/models/user.py

```diff
@@ -34,166 +34,13 @@ class User(AbstractUser):
     LOGIN_FIELDS = ("username", "email")
 
     email = models.EmailField(_("email address"), unique=True)
-    is_demo = models.BooleanField(default=False, help_text="Is a demo account?")
```

Comment:
Maybe we could delete all these accounts before removing the field?

Reply:
Note, that I'm not sure the field is set to True for any account and it may be set to True mistakenly.

Reply:
I confirm it's not used in production data